project_test/lib/__init__.py

```diff
@@ -0,0 +1,3 @@
+#from .icr_tools import ICRTools
+from .message_filters import MessageFilters
+from .pipeline_configurations import PipelineConfiguration
```

Comment:
No new line at end of file.

There will be a switch in your editor that forces files to end with a new line.